src/contexts/UserDataContext.jsx

```diff
@@ -22,9 +23,11 @@ const UserDataContextProvider = ({ children }) => (
   <SignedInUserContextProvider>
     <SelectedUserContextProvider>
       <UserListContextProvider>
-        <DocumentListContextProvider>
-          <MessageContextProvider>{children}</MessageContextProvider>
-        </DocumentListContextProvider>
+        <UserDocumentListContextProvider>
```

Comment:
What's the difference between UserDocumentContext and ClientDocumentContext? They seem to be copies of eachother.

Reply:
Yeah, one is based on `selectedUser` and the other is based on `podUrl`.

Making separate contexts for the user's document list and the client's document list.

Reply:
I'm not sure how useful this distinction is. There's no situation where the user will see more than one set of documents at a time. If the user wants to change to viewing documents from a different pod, the documents context can clear out and load a different document list.

This code was working before and covered these two cases without this distinction, so I don't see the value in adding it.

Reply:
I'll revert this for now.